README.md

```diff
@@ -1,45 +1,50 @@
 
-[![pipeline status](https://gitlab.com/meltano/meltano/badges/master/pipeline.svg)](https://gitlab.com/meltano/meltano/commits/master)
+Meltano
+-------
 
-[Meltano](https://meltano.com) is
-[open source](https://gitlab.com/meltano/meltano),
-[self-hosted](https://docs.meltano.com/guide/production),
-[CLI-first](https://docs.meltano.com/reference/command-line-interface),
-[debuggable](https://docs.meltano.com/reference/command-line-interface#debugging), and
-[extensible](https://docs.meltano.com/concepts/plugins).
+Welcome to the open source DataOps Infrastructure to manage and centralize all the data tools in your stack and turn them into your ideal end-to-end data platform. With Meltano, you'll be able to collaboratively build and improve your data platform like a software project; spinning up a service or tool (Airflow, dbt, Great Expectations, Snowflake, etc) and easily configuring, deploying and managing it through a single control plane.
 
-[Pipelines are code](https://docs.meltano.com/concepts/project),
-ready to be version controlled,
-[containerized](https://docs.meltano.com/guide/containerization), and
-[deployed continuously](https://docs.meltano.com/guide/production#your-meltano-project).
-Develop and test
-[locally](https://docs.meltano.com/getting-started#install-meltano),
-then
-[deploy in production](https://docs.meltano.com/guide/production)
-along with the built-in
-[Airflow integration](https://docs.meltano.com/guide/production#airflow-orchestrator),
-or inside your
-[orchestrator of choice](https://docs.meltano.com/guide/production#running-pipelines).
+![](https://lh3.googleusercontent.com/v77ESPe2YLNxpfZNv9zYIle0vd59Oe4gr5iOMzhHamo_dwMuIxrF8zsVhodOc87xRsA6VXt-ZD5saL7MD4f0bmLzwrptUp_JtmSvKjicjBAmgmu_YtSz3PrxvJ-VvvN63mpgVGdiyLM7DPgbaw)
 
-Meltano [embraces](https://handbook.meltano.com/product/singer) the [Singer](https://www.singer.io/) standard and its community-maintained library of open source
-[extractors](https://hub.meltano.com/extractors/) and
-[loaders](https://hub.meltano.com/loaders/),
-and leverages [dbt](https://www.getdbt.com) for [transformation](https://docs.meltano.com/guide/transformation).
+Integrations
+------------
 
-## Documentation
+[MeltanoHub](https://hub.meltano.com/) is the single source of truth to find any Meltano plugins as well as [Singer](https://singer.io/) taps and targets. Users are also able to add more plugins to the Hub and have them immediately discoverable and usable within Meltano. The Hub is lovingly curated by Meltano and the wider Singer community.
 
-Check out the ["Getting Started" guide](https://docs.meltano.com/getting-started)
-or find the full documentation at <https://docs.meltano.com>.
+Installation
+------------
 
-## Contributing to Meltano
+If you're ready to run an EL(T) pipeline with a data source and destination of your choosing, start by following the [Installation guide](https://docs.meltano.com/guide/installation) to have Meltano up and running in your device.
```

Comment:
What about something like "If you're ready to build your ideal data platform and start running data workflows, start by..."

```suggestion
If you're ready to build your ideal data platform and start running data workflows across multiple tools, start by following the [Installation guide](https://docs.meltano.com/guide/installation) to have Meltano up and running in your device.
```